requirements.txt

```diff
@@ -0,0 +1,3 @@
+ws4py
+requests
+pygame
```

Comment:
Also needs pytest.

Reply:
Take a look at my upcoming branch called "easier_config".

There is a setup.py file and updated documentation.

It would be nice to know if this approach is easier or harder for you.